torch/_torch_docs.py

```diff
@@ -2362,10 +2362,22 @@ def merge_dicts(*dicts):
 
 .. math::
     \text{out} = \text{abs} \cdot \cos(\text{angle}) + \text{abs} \cdot \sin(\text{angle}) \cdot j
+
+The behavior of this function is undefined if the absolute value is NaN, or if the angle is infinite.
```

Comment:
I would move these paragraphs into the note

Reply:
Done